discovery/gossiper.go

```diff
@@ -1178,14 +1185,12 @@ func (d *AuthenticatedGossiper) retransmitStaleChannels() error {
 			return nil
 		}
 
-		const broadcastInterval = time.Hour * 24
-
 		timeElapsed := time.Since(edge.LastUpdate)
 
 		// If it's been a full day since we've re-broadcasted the
 		// channel, add the channel to the set of edges we need to
 		// update.
-		if timeElapsed >= broadcastInterval {
+		if timeElapsed >= d.cfg.RebroadcastInterval {
```

Comment:
Nit: update comment since rebroadcast interval is now configurable.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2084/commits/50980662ef58fdab9bd5e1215ccb97bf52d9fd00